### ISSUE TITLE
Does the pacakge offer APIs?

### ISSUE THREAD

**Issue by sprklinginfo** [thekordy/ticketit#576]
Hi there, firstly thank you for your amazing work, especially it can be included in an existing project. It looks like only a logged-in user can submit tickets. Does the package offer APIs to create/manage tickets? The use case will be a visitor fills a form and submit it and behind the scene, it will call the API to create a ticket. Thanks.

**Comment by balping** [comment]
No, sorry, not currently. It theory it's planned, but there hasn't been any work on the new version for years.

**Comment by JT1-1** [comment]
hello I have a question, I add a ticket but it is not displayed .. it is saved in the database
